client/blocks/reader-feed-item/index.tsx

```diff
@@ -0,0 +1,240 @@
+import { recordTrainTracksInteract } from '@automattic/calypso-analytics';
```

Comment:
- Refactored `ReaderUnsubscribedFeedItem` to `ReaderFeedItem` which now handles both wpcom and non-wpcom feeds. Now we have one component instead of 3 and with much easier props.

- Simplified the properties that we need to display in HTML markup. Previously we were using 2 different components to pass these properties but now we are getting these properties from proper place by using `isWpcomFeed` as conditional.

- The tracks implementation was almost duplicated in `ReaderUnsubscribedWpcomFeedItem` & `ReaderUnsubscribedNonWpcomFeedItem` so used it as is.

- HTML markup is same as before.